laravel-app/app/Http/Controllers/FollowController.php

```diff
@@ -20,27 +20,33 @@ public function show(User $user): View
         ]);
     }
 
-    public function update(User $user): RedirectResponse
+    public function update(User $user): JsonResponse
```

Comment:
Add comment!

Reply:
done sir, added function comment. Thank you